frb_codegen/src/config/opts_parser.rs

```diff
@@ -172,12 +233,66 @@ pub fn config_parse(mut raw: RawOpts) -> Vec<Opts> {
                 wasm_enabled: wasm,
                 dart3,
                 inline_rust,
+                shared: false,
+                shared_rust_output_path: shared_rust_output_path.clone(),
+                shared_dart_output_path: shared_dart_output_path.clone(),
                 bridge_in_method,
                 keep_going,
                 extra_headers: extra_headers.clone(),
             }
         })
-        .collect()
+        .collect::<Vec<_>>();
+
+    // if shared API-block(config) is essential, generate and add it to vec
+    assert!(!regular_configs.is_empty());
+    let all_configs = if regular_configs.len() == 1 {
+        regular_configs.clone()
+    } else {
+        // NOTE: since there would be at least 1 shared API called `free_WireSyncReturn` for multi-blocks,
+        // the extra shared config is essential, whatever `_implicit_duplicated_symbols` is empty or not.
```

Comment:
nit: I quickly searched `_implicit_duplicated_symbols` without matches in this file...

Reply:
Fixed in [6e91e83](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145/commits/6e91e83e38d39da174a5856935cb96c91078d88e).